### PR TITLE
Post PHPSC changes

### PR DESCRIPTION
**Pull request by jezemery**
Commits should explain, but removing PHP Hampshire CIC from footer and removing donate button are the major changes here.

### CODE REVIEW

File: public-src/phphants/scss/style.scss [hunk 1]
```diff
@@ -264,16 +264,20 @@ header .login-status-text {
     cursor: pointer;
 }
 
-.content{
-    flex: 1 0 auto;
+.content {
+    display: flex;
```
Comment (lboynton) [inline public-src/phphants/scss/style.scss]: Er I think this will cause a problem in IE, see #157

Comment (lboynton) [inline public-src/phphants/scss/style.scss]: It might work fine. Worth checking anyway!

Comment (jezemery) [inline public-src/phphants/scss/style.scss]: Hmm. This says support is ok...
screen shot 2017-06-12 at 20 18 42

I don't have a windows machine handy to test on, so I'll have to put this on hold

Comment (lboynton) [inline public-src/phphants/scss/style.scss]: https://saucelabs.com/open-source ?

Comment (lboynton) [inline public-src/phphants/scss/style.scss]: But as you can see, IE only has partial support due to various bugs

**Comment by jezemery** [comment]
In which case I'll close this for now

**Pull request closed without merging**